README.md

```diff
@@ -195,7 +195,7 @@ run
 # invalid
 ```
 
-#### `@Command.String(optionNames: string, {tolerateBoolean?: boolean})`
+#### `@Command.String(optionNames: string, {tolerateBoolean?: boolean, description?: string})`
```

Comment:
Not sure if you want me to add it here as well? Some other options like `hidden` or `tolerateBoolean` are not defined here as well. Might've been forgotten in the past.